src/main/webapp/scripts/fetchData.js

```diff
@@ -33,18 +39,55 @@ function fetchSpeciesData(name) {
 
         // Update description entry
         var notes = speciesData.wikipediaNotes;
-        descriptionContainer.innerText = notes == null ? "N/A" : notes.substr(0, notes.indexOf('['));
+        switch(notes) {
+            case null:
+                descriptionContainer.innerText = "N/A";
+                break;
+            case "":
+                descriptionContainer.innerText = "N/A";
+                break;
+            default:
+                if (notes.indexOf('[') > 0) {
+                    descriptionContainer.innerText = notes.substr(0, notes.indexOf('['));
```

Comment:
Future note: Since we'll be cleaning wikipedia notes to remove the parenthesis, you won't need this anymore